libraries/AP_XRCE_Client/Idl/BatteryState.idl

```diff
@@ -9,31 +9,31 @@ module sensor_msgs {
     module BatteryState_Constants {
       @verbatim (language="comment", text=
         "Constants are chosen to match the enums in the linux kernel" "\n"        "defined in include/linux/power_supply.h as of version 3.7" "\n"        "The one difference is for style reasons the constants are" "\n"        "all uppercase not mixed case." "\n"        "Power supply status constants")
-      const uint8 POWER_SUPPLY_STATUS_UNKNOWN = 0;
```

Comment:
Annoying this is even a thing, the constants don't even show up as part of the generated header.